kolibri/plugins/learn/assets/src/views/SearchPage.vue

```diff
@@ -74,6 +74,21 @@
     computed: {
       ...mapState('search', ['contents', 'searchTerm', 'total_results']),
     },
+    beforeDestroy() {
+      // TODO do this clean up in a beforeRouteLeave once SearchPage is rendered in router-link
```

Comment:
is this essentially the same as what's proposed in https://github.com/learningequality/kolibri/issues/5719 ?

Reply:
Yeah, implementing #5719 will give us access to those vue router hooks in this page

Reply:
There’s also some other similar vuex module cleanup code that is run in a central router hook that could be potentially moved to a router hook in the same way